packages/component-library-react/src/FormFieldTextbox.test.tsx

```diff
@@ -0,0 +1,881 @@
+
+    expect(richText).toBeInTheDocument();
+  });
+
+  it('can be hidden', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} hidden />);
+
+    const field = container.querySelector('div');
+
+    expect(field).not.toBeVisible();
+  });
+
+  it('can have a custom class name', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} className="invalid" />);
+
+    const field = container.querySelector('div');
+
+    expect(field).toHaveClass('invalid');
+  });
+
+  it('can have a additional class name', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} className="large" />);
+
+    const field = container.querySelector(':only-child');
+
+    expect(field).toHaveClass('large');
+    expect(field).toHaveClass('utrecht-form-field');
+  });
+
+  describe('label', () => {
+    it('renders a design system BEM class name: utrecht-form-field__label', () => {
+      const { container } = render(<FormFieldTextbox {...defaultProps} />);
+
+      const field = container.querySelector('.utrecht-form-field__label');
+
+      expect(field).toBeInTheDocument();
+    });
+
+    it('renders rich text content', () => {
+      const { container } = render(
+        <FormFieldTextbox
+          {...defaultProps}
+          label={
+            <>
+              I can speak the <i lang="la">lingua franca</i>
+            </>
+          }
+        />,
+      );
+
+      const richText = container.querySelector('i');
+
+      expect(richText).toBeInTheDocument();
+    });
+
+    it('is associated with the textbox', () => {
+      render(<FormFieldTextbox {...defaultProps} />);
+
+      const textbox = screen.getByRole('textbox', { name: 'Subject' });
+
+      expect(textbox).toBeInTheDocument();
+    });
+
+    it.skip('can be clicked to focus the textbox', () => {
+      const handleFocus = jest.fn();
+      const labelRef = createRef<HTMLSpanElement>();
+
+      render(<FormFieldTextbox {...defaultProps} label={<span ref={labelRef}>Subject</span>} onFocus={handleFocus} />);
+
+      //   const textbox = screen.getByRole('textbox', { name: 'Subject' });
+
+      labelRef.current?.click();
+
+      expect(handleFocus).toHaveBeenCalled();
+    });
+  });
+
+  describe('description', () => {
+    it('is not rendered by default', () => {
+      const { container } = render(<FormFieldTextbox {...defaultProps} />);
+
+      const field = container.querySelector('.utrecht-form-field__description');
+
+      expect(field).not.toBeInTheDocument();
+    });
+
+    it('renders a design system BEM class name: utrecht-form-field__description', () => {
+      const { container } = render(
+        <FormFieldTextbox
+          {...defaultProps}
+          description="Lingua franca is a common language between groups of people."
+        />,
+      );
+
+      const field = container.querySelector('.utrecht-form-field__description');
+
+      expect(field).toBeInTheDocument();
```

Comment:
I would love to also check here the description string is the content of the queried `field`